faq.md

```diff
@@ -61,6 +62,22 @@ Request compressRequest(Request request) {
 Future<Response> postRequest(@Body() Map<String, String> data);
 ```
 
+## Runtime baseUrl change
+
+Sometimes you may need to change runtime base url, for example, you have server selector and you want to use different api routes like
+`regular` and `pro` routes. You can store the current server base url in your SharedPreferences (encrypt/decrypt it if needed) and
+use in interceptor like that:
```

Comment:
Why is this a new line? :)

Reply:
Ooops, I'm dumb :D Just wrapped textline in VS Code :D